packages/containers/sandbox/src/app/components/Showcase/Showcase.container.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { Layout } from '@talend/react-components';
+import { Inject } from '@talend/react-cmf';
+
+import EmptyShowcase from './EmptyShowcase';
+import theme from './Showcase.scss';
+
+const Showcase = ({ content }) => {
+	const child = content.component === 'EmptyShowcase'
+		? <EmptyShowcase />
+		: <Inject {...content} />;
```

Comment:
Should be a function with a regular if/else statement.

Reply:
It don't see why, could you elaborate ?

Reply:
Words convey much more meaning about your intention with the code than signs.

In your case someone how reads the code has to decipher the meaning of four sign `=`, `===`, `?`, `:` and there impact on the computation of the constant. Whereas using a properly named function will convey the same meaning without requiring the reader to truly understand the inner working of the function. And even if he/she is interested as to what happens inside the function its purpose is expressed through a single `if` statement. 

And on top of that it makes it trivial to write a unit test.

Reply:
I would assume that someone who would come to read this would know how to code, thus reading this ternary.
Also, it wouldn't make a difference on unit tests since this method will not be exported (the only point to export it would be testing it).
But as you wish ;)